app/assets/javascripts/discourse/testem.js

```diff
@@ -127,14 +127,13 @@ class Reporter extends TapReporter {
 module.exports = {
   test_page: "tests/index.html?hidepassed",
   disable_watching: true,
-  launch_in_ci: ["Chrome"],
-  // launch_in_dev: ["Chrome"] // Ember-CLI always launches testem in 'CI' mode
+  launch_in_ci: ["Chromium"],
```

Comment:
IIRC this config option is badly named, and ember-cli runs testem in 'CI' mode, even in local development.

Reply:
Their docs say `launch_in_ci:  [Array]  list of launchers to use for CI runs`, so I guess this is a list of launchers to use if none is specified maybe? I’m not familiar at all with testem :sweat_smile: